content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -280,65 +267,56 @@ Here is a Code snippet to illustrate the ```extends``` keyword:
 public class MainDog extends Dog {
  
  //abstract method body definition
- 
+ @Override
  public void run(){
- 
-     //in this class the method is implemented
- 
-     System.out.println(“Dog is running”);
+   //in this class the method is implemented
+    System.out.println(“Dog is running”);
  
  }
  
  public static void main(Strings [] args){
- 
-     MainDog dog = new MainDog();
- 
-     //abstract method invocation
- 
-     dog.run();
- 
- }
+    
+        MainDog dog = new MainDog();
+    
+        //abstract method invocation
+    
+        dog.run();
+    
+    }
  
 }
  
 ```
  
 #### Interface implementation
-An interface contains public, abstract method signatures (methods without implementation). It is also important to note that interfaces have no constructors and may contain final and static variables. Implementating abstraction by the interface method is possible by the use of the `implements` keyword.
+An interface contains public, abstract method signatures (methods without implementation). It is also important to note that interfaces have no constructors and may contain final and static variables. Implementating abstraction by the interface method is possible by the use of the `implements` keyword. Since methods in interface do not have body, the class implementing it must implement all the methods in the interface.
```

Comment:
> Since methods in interface do not have body, the class implementing it must implement all the methods in the interface.

change this to:

> Since methods in the interface do not have a body, the class implementing it must implement all the methods in the interface.

As suggested by Grammarly. Always make sure to check your sentences with [Grammarly](https://app.grammarly.com/) before committing. Since the words *interface* and *body* are [common nouns](https://www.grammarly.com/blog/common-noun/), you need to an [**article word**](https://www.scribendi.com/academy/articles/using_articles_a_an_the.en.html), i.e., *a*, *an* or *the*.

Reply:
Done